app.py

```diff
@@ -108,6 +178,60 @@ def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
         bucket_arn=website.bucket.bucket_arn,
     )
 
+db_secret_name = database.pgstac.secret.secret_name
+db_security_group = database.db_security_group
+
+# ingestor config requires references to other resources, but can be shared between ingest api and bulk ingestor
+ingestor_config = ingest_config(
+    stac_db_vpc_id=vpc.vpc.vpc_id,
+    stac_db_secret_name=db_secret_name,
+    stac_db_security_group_id=db_security_group.security_group_id,
+    stac_db_public_subnet=database.is_publicly_accessible,
+    stac_api_url=stac_api.stac_api.url,
+    raster_api_url=raster_api.raster_api.url,
+)
+
+env_secret = veda_stack.build_env_secret(
```

Comment:
The environment configuration parameters should not be updated by CDK deploy. The build env secret method would be a super useful utility function in support_scripts to facilitate setting up a secret for a new instance.